style.css

```diff
@@ -8,6 +8,17 @@ body {
     flex-direction: column;
     align-items: center;
 }
+ li::after {
+     content: '';
+     display: block;
+     width: 0;
+     height: 2px;
+     background: rgb(151, 74, 223);
+     transition: width 0.3;
+ }
+ li:hover::after {
+    width: 100%;
+}
 
 a {
     text-decoration: none;
```

Comment:
I think we can take out the a { text-decoration: none; }, after fixing the merge conflict these list items are no longer links. I just gave them a pointer cursor since we don't have any pages to link them to for now. My only other suggestion would be to add the nav-list-top class in front of the li in the css. That way we won't have to worry about the hover effect being applied to list items where we don't want it applied. So I would change it to this: 

` .nav-list-top li::after {
     content: '';
     display: block;
     width: 0;
     height: 2px;
     background: rgb(151, 74, 223);
     transition: width 0.3;
 }
 .nav-list-top li:hover::after {
    width: 100%;
}`

Other than that I think this looks great! I fixed a couple merge conflicts that I think were just a result of your code not being updated with some changes I made. If you do 'git pull' before you start working on a new feature it will make sure you are up to date with the main branch. :)

Reply:
Awesome! So I fixed the hover effect after I committed because I noticed the error after. Also thanks for the heads up on doing git pull first. :)